models.go

```diff
@@ -62,6 +63,8 @@ type AppPermission struct {
 	RequestMethod           string  `gorm:"index" validate:"required"`
 	MaxAmount               int
 	MaxAmountPerTransaction int
+	BudgetType              string
```

Comment:
I think BudgetType is not very explanatory. BudgetRenewal (what it is called in the UI seems better?)